package.json

```diff
@@ -4,12 +4,16 @@
   "private": true,
   "dependencies": {
     "@fortawesome/react-fontawesome": "^0.2.0",
+    "@reduxjs/toolkit": "^1.9.0",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-redux": "^8.0.5",
+    "react-router-dom": "^6.4.3",
     "react-scripts": "5.0.1",
+    "react-test-renderer": "^18.2.0",
     "web-vitals": "^2.1.4"
```

Comment:
- [ ] Checking through your package.json file, I have realized that you have not included the Redux package (It is not listed here in the list of dependencies). As it is a requirement in the project, you are required to use React together with Redux in your project. Kindly include it.
- [ ] This also explains why you do not have a redux directory in your project and instead you are having a features folder. To meet all the requirements, you require to have  Redux and use it in your project to manage your state.

**This is an example of a project using Redux to manage its state in their React app**

![image](https://user-images.githubusercontent.com/86318284/203925595-80686d38-b397-4422-8239-8377555c147c.png)